benchmark/sparse/linear_regression/results.md

```diff
@@ -0,0 +1,27 @@
+# Linear Regression Benchmark Results 
+
+## Summary
+```
+   Results below show the performance comparison of linear regression with MXNet vs Keras-Tensorflow using sparse tensors
+```
+
+### Configuration
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.0                                                       |
+| TensorFlow       | v1.9.0                                                      |
+| MXNet            | v1.2.1                                                       |
+
+
+### Results 
+| Instance Type | GPUs  | Batch Size  | MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
+|-----|-----|-----|----- |-----|
+|  C5.18X Large |   0  | 16  | 51.15 sec | 71.85 sec |
+|  C5.18X Large |   0  | 32 | 20.82 sec | 54.31 sec |
+|  C5.18X Large |   0  | 64  | 13.13 sec | 19.20 sec |
+|  C5.18X Large |   0  | 128  | 5.72 sec | 9.86 sec |
+
+### Note
+For reproducing above results set seed to `7` by adding this line in the `run_sparse_benchmark` script - `np.random.seed(7)`
```

Comment:
why don't we add this line rather than asking the user to add.

Reply:
We wanted to avoid the assumption that this benchmark is reproducible only one one dataset. So as per comments by @roywei we moved it under ReadMe - for details about how to get these values.

Reply:
Or we can report an average of multiple runs. My point is better performance on a fixed seed is not convinced enough.